gdpr/management/commands/deactivate_expired_reasons.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand
+
+from gdpr.models import LegalReason
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+
+        for legal_reason in LegalReason.objects.filter_expired_retaining_data_in_last_days():
+            legal_reason._expirement()
```

Comment:
why legal_reason._expirement() and not legal_reason.expire()?

Reply:
Oj, ano, diky. Opraveno.

Reply:
Tak ne, je to spravne z toho duvodu, ze v tomto pripade nechceme zmenit datum vyprseni duvodu.